### ISSUE TITLE
Surface context on the workspace start page when creating a new or opening an existing workspace

### ISSUE THREAD

**Issue by gtsiolis** [gitpod-io/gitpod#13712]
## Problem to solve

Currently, when users create a new workspace or open an existing workspace, they go through the **workspace start** page (Checking, Creating, etc) without surfacing the original context.

Sometimes, opening a workspace can take up to 5 minutes and users can be lost as we're showing a generic page for **_any_** context or repository they open.

### Proposal

There are several improvements we could make on the workspace start page (see [relevant discussion](https://gitpod.slack.com/archives/C01KPEPQYE6/p1658931696736879) (internal)), but a good MVC (minimal viable change) here could be to simply reuse the context element as we do with the rest of the workspace start phases (`STOPPED`, `TIMED-OUT`, etc.).

**Comment by Hardik500** [comment]
@gtsiolis I would like to work on this issue. Is this up for grab, or some discussion is required on this one?

**Comment by gtsiolis** [comment]
@Hardik500 Yes, using the context element should be ok to add here. Let us know if you open a PR for resolving this.
